test/ModuleTest.php

```
@@ -117,10 +118,10 @@ public function expectedListeners(): array
     }
 
     /**
-     * @dataProvider expectedListeners
      * @psalm-param callable(MvcEvent|MvcAuthEvent):null|Response $listener
      * @psalm-param MvcEvent::EVENT_*|MvcAuthEvent::EVENT_* $event
      */
+    #[DataProvider('expectedListeners')]
```

PHPStan (2.2.7)

[error] PHPDoc tag `@param` for parameter $listener with type (callable)|Laminas\Http\Response is not subtype of native type callable.


(parameter.phpDocType)